runtime/vm/classsupport.c

```diff
@@ -307,11 +307,19 @@ internalFindClassString(J9VMThread* currentThread, j9object_t moduleName, j9obje
 		char localBuf[J9VM_PACKAGE_NAME_BUFFER_LENGTH];
 		char *utf8Name = NULL;
```

Comment:
For consistency with the other code, please declare this as `U_8 *` and cast it on the assignment, rather than casting the uses.

Reply:
Updated the code involved against the suggestion above.